ggshield/core/git_shell.py

```diff
@@ -82,6 +82,17 @@ def git(
             cwd=cwd,
         )
         return result.stdout.decode("utf-8", errors="ignore").rstrip()
+    except subprocess.CalledProcessError as e:
+        if "detected dubious ownership in repository" in e.stderr.decode(
+            "utf-8", errors="ignore"
+        ):
+            raise UnexpectedError(
+                "Git command failed because of a dubious ownership in repository.\n"
+                "If you still want to run ggshield, make sure you mark "
+                "the current repository as safe for git.\n"
+                "`git config --global --add safe.repository=<YOUR_REPO>`"
```

Comment:
```suggestion
                "the current repository as safe for git with:\n\n"
                "    git config --global --add safe.repository <YOUR_REPO>"
```

I think it would look a bit nicer with the indent I added.

Before:
```
$ ggshield secret scan repo .        
Error: Git command failed because of a dubious ownership in repository.
If you still want to run ggshield, make sure you mark the current repository as safe for git.
`git config --global --add safe.repository=<YOUR_REPO>`
```

After:
```
$ ggshield secret scan repo .                   
Error: Git command failed because of a dubious ownership in repository.
If you still want to run ggshield, make sure you mark the current repository as safe for git with:

    git config --global --add safe.repository <YOUR_REPO>
```

More important: I removed the `=` before `<YOUR_REPO>`: the command fails otherwise.

Reply:
Oops, nice catch ! Thanks !